packages/emmett-expressjs/src/testing/apiSpecification.ts

```diff
@@ -59,9 +61,9 @@ export const expectNewEvents = <EventType extends Event = Event>(
 };
 
 export const expectResponse =
-  (
+  <Body = unknown>(
```

Comment:
A small addition to the `expectResponse` matcher so we can optionally type the body

Reply:
Makes perfect sense to me, thanks for doing that! 🫡